WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -0,0 +1,28 @@
+import UITestsFoundation
+import XCTest
+
+class DashboardTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        setUpTestSuite()
+
+        try LoginFlow.login(
+            siteUrl: WPUITestCredentials.testWPcomSiteAddress,
+            email: WPUITestCredentials.testWPcomUserEmail,
+            password: WPUITestCredentials.testWPcomPassword
+        )
+    }
+
+    override func tearDownWithError() throws {
+        takeScreenshotOfFailedTest()
+        removeApp()
+    }
+
+    func testDomainsCardNavigation() throws {
+        try MySiteScreen()
+            .verifyDomainsCard()
+            .tapDomainsCard()
+
+        XCTAssertTrue(DomainsScreen.isLoaded(), "\"Domains\" screen isn't loaded after \"Domains\" card tap.")
```

Comment:
what do you think about moving this to a `verifyDomainsScreenLoaded` function so we could chain it in the test? i see it's not the standard way for tests in the repo (i think i got too used to seeing it on WCiOS 😄 ) but i think the test reads better that way

Reply:
Good suggestion @jostnes 👍 Done in 144637bd61211d7a62532eeca7d70f2b5a68a6c3.